treeherder/config/settings.py

```diff
@@ -327,76 +399,73 @@
     },
 }
 
-# rest-framework settings
+# CORS Headers
+CORS_ORIGIN_ALLOW_ALL = True  # allow requests from any host
+
+# Debug Toolbar
+if ENABLE_DEBUG_TOOLBAR:
+    # django-debug-toolbar requires that not only DEBUG be set, but that the request IP
+    # be in Django's INTERNAL_IPS setting. When using Vagrant, requests don't come from localhost:
+    # http://blog.joshcrompton.com/2014/01/how-to-make-django-debug-toolbar-display-when-using-vagrant/
+    # If the Vagrant IPs vary by platform or if there isn't a consistent IP when we switch to Docker,
+    # we'll have to do: https://github.com/jazzband/django-debug-toolbar/pull/805#issuecomment-240976813
+    INTERNAL_IPS = ['127.0.0.1', '10.0.2.2']
+
+if ENABLE_DEBUG_TOOLBAR:
```

Comment:
This conditional is duplicated above. Also, in the Django section immediately after `INSTALLED_APPS` there is an `if DEBUG` that adds the same app. I think the reason we used the additional environment variable before, was that django debug toolbar is in `requirements/dev.txt`, so if `DEBUG` was enabled on eg prototype for some reason, then the dependency wouldn't be found, and so the app not start. However I really doubt using DEBUG is something we ever would need to (or should) do, even on the prototype instance -- so let's just ditch `ENABLE_DEBUG_TOOLBAR` (incl from `env.sh`).